src/lucky_cli/task.cr

```diff
@@ -7,6 +7,19 @@ abstract class LuckyCli::Task
     def name
       "{{@type.name.gsub(/::/, ".").underscore}}"
     end
+
+    def help_message
+      "Run this task with `lucky #{name}`"
```

Comment:
I think we tend to use single quotes instead of backpacks in most places in the help messages in Lucky